openghg/analyse/_scenario.py

```diff
@@ -0,0 +1,1453 @@
+            )
+
+        if output_TS:
+            timeseries = DataArray(timeseries, dims=("time"), coords={"time": time_array})
+
+        if output_fpXflux and output_TS:
+            timeseries.compute()
+            fpXflux.compute()
+            return timeseries, fpXflux
+        elif output_fpXflux:
+            fpXflux.compute()
+            return fpXflux
+        elif output_TS:
+            timeseries.compute()
+            return timeseries
+
+        return None
+
+    def footprints_data_merge(self,
+                              resample_to: str = "coarsest",
+                              platform: Optional[str] = None,
+                              calc_timeseries: bool = True,
+                              sources: Optional[Union[str, List]] = None,
+                              calc_bc: bool = True,
+                              cache: bool = True,
+                              recalculate: bool = False) -> Dataset:
+        """
+        Produce combined object containing aligned footprint and observation data.
+        Can also include modelled timeseries data derived from flux.
+
+        Args:
+            resample_to: Resample option to use for averaging:
+                          - either one of ["coarsest", "obs", "footprint"] to match to the datasets
+                          - or using a valid pandas resample period e.g. "2H".
+                         Default = "coarsest".
+            platform: Observation platform used to decide whether to resample.
+            calc_timeseries: Calculate modelled timeseries based on flux sources.
+            sources: Sources to use for flux if calc_timseries is True.
+                     All will be used and stacked if not specified.
+            calc_bc: Calculate boundary conditions (not currently implemented)
+            cache: Cache this data after calculation. Default = True.
+            recalculate: Make sure to recalculate this data rather than return from cache. Default = False.
+
+        Returns:
+            xarray.Dataset: Combined dataset containing footprint and observation data
+        """
+        combined_dataset = self.combine_obs_footprint(resample_to=resample_to,
+                                                      platform=platform,
+                                                      cache=cache,
+                                                      recalculate=recalculate)
+
+        if calc_timeseries:
+            modelled_obs = self.calc_modelled_obs(resample_to=resample_to,
+                                                  sources=sources,
+                                                  platform=platform,
+                                                  cache=cache,
+                                                  recalculate=recalculate)
+
+            name = modelled_obs.name
+            combined_dataset = combined_dataset.assign({name: modelled_obs})
+
+        if calc_bc:
```

Comment:
Should we maybe print a warning here saying they're not implemented yet?

Reply:
Yes I can add that